tests/unit/NameRequestInfo.spec.ts

```diff
@@ -0,0 +1,184 @@
+
+Vue.use(Vuetify)
+
+const vuetify = new Vuetify({})
+const store = getVuexStore() as any // remove typings for unit tests
+
+const localVue = createLocalVue()
+localVue.filter('VMask', data => data) // mock filter
+localVue.filter('capitalize', data => data) // mock filter
+
+const approvedCpNamerequest = {
+  nrNum: 'NR 1234567',
+  legalType: 'CP',
+  filingId: 0,
+  applicants: {
+    // address info
+    addrLine1: '1012 Douglas St',
+    addrLine2: 'Suite 200',
+    addrLine3: 'Second Floor',
+    city: 'Victoria',
+    stateProvinceCd: 'BC',
+    postalCd: 'V8W 2C3',
+    countryTypeCd: 'CA',
+    // contact info
+    emailAddress: 'email@example.com',
+    phoneNumber: '(123) 456-7890',
+    // name info
+    firstName: 'First',
+    middleName: 'Middle',
+    lastName: 'Last'
+  },
+  names: [
+    {
+      state: 'APPROVED',
+      name: 'Approved CP Namerequest'
+    }
+  ],
+  consentFlag: null,
+  expirationDate: '2022-07-05T06:59:00+00:00',
+  state: 'APPROVED'
+}
+
+const conditionalSpNamerequest = {
+  nrNum: 'NR 1234567',
+  legalType: 'SP',
+  filingId: 0,
+  applicants: {
+    // address info
+    addrLine1: '1012 Douglas St',
+    addrLine2: 'Suite 200',
+    addrLine3: 'Second Floor',
+    city: 'Victoria',
+    stateProvinceCd: 'BC',
+    postalCd: 'V8W 2C3',
+    countryTypeCd: 'CA',
+    // contact info
+    emailAddress: 'email@example.com',
+    phoneNumber: '(123) 456-7890',
+    // name info
+    firstName: 'First',
+    middleName: 'Middle',
+    lastName: 'Last'
+  },
+  names: [
+    {
+      state: 'CONDITION',
+      name: 'Conditional SP Namerequest'
+    }
+  ],
+  consentFlag: 'X',
+  expirationDate: '2021-07-05T06:59:00+00:00',
+  state: 'EXPIRED'
+}
+
+describe('NameRequestInfo component', () => {
+  it('renders an approved coop namerequest correctly', async () => {
+    store.state.entityType = 'CP'
+    const wrapper = mount(NameRequestInfo,
+      {
+        propsData: { nameRequest: approvedCpNamerequest },
+        store,
+        localVue,
+        vuetify
+      })
+    await Vue.nextTick()
+
+    // verify first column
+    const firstCol = wrapper.findAll('.col').at(0)
+    expect(firstCol.find('.hdr').text()).toBe('Name Request')
+
+    let keys = firstCol.findAll('.key')
+    expect(keys.at(0).text()).toBe('Entity Type:')
+    expect(keys.at(1).text()).toBe('Request Type:')
+    expect(keys.at(2).text()).toBe('Expiry Date:')
+    expect(keys.at(3).text()).toBe('Status:')
+    expect(keys.at(4).text()).toBe('Condition/Consent:')
+
+    let vals = firstCol.findAll('.val')
+    expect(vals.at(0).text()).toBe('BC Cooperative Association')
+    expect(vals.at(1).text()).toBe('New Business')
+    expect(vals.at(2).text()).toBe('July 4, 2022 at 11:59 pm Pacific time')
+    expect(vals.at(3).text()).toBe('APPROVED')
+    expect(vals.at(4).text()).toBe('Not Required')
+
+    expect(firstCol.find('#status .nr-status-icon.mdi-check').exists()).toBe(false)
```

Comment:
No, you shouldn't have to do this (but your codebase doesn't have the latest NameRequestInfo component).

If possible, remove this file from your PR. It might actually be easier to close this PR and start a new one?

Reply:
> 1. if possible, don't check in the new unit test file
> 2. change this PR's target to release-5.0.4 branch

I've changed the base to release-5.0.4 branch and removed NameRequestInfo.spec.ts file. I hope it's okay, or I will close this one and start a new one.

Reply:
Looks good! I'll merge it when the checks finish.